macro/Version.py

```diff
@@ -47,11 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'm-turtle':
-            # TODO Change when m-turtle name is decided
+        if version.lower() == 'melodic':
             return ('<span style="background-color:#FFFF00; '
                     'font-weight:bold; padding: 3px;">'
-                    'Expected in M-Turtle</span>')
+                    'Expected in Melodic Morenia</span>')
```

Comment:
I have looked through the history the condition in https://github.com/ros-infrastructure/roswiki/blob/9cec5a32d35d430691e6bfd3e58852cae2604775/macro/Version.py#L50 and it has always referenced the ROS distro name (rather than *x*-turtle). I think that makes more sense compared to switching between the two ways. Also the ROS distro name should always be known for the next distro when a new one is released. Therefore I think it should be changed to `melodic` (and if any pages already use `m-turtle` they should be updated).